_steps/level0/step0-tools.markdown

```diff
@@ -49,7 +47,7 @@ Dev Tools provide you a way to see what is going on behind the scenes in whateve
 Browser Dev Tools are useful for:
 
 1. Trying to figure out why your page isn't doing the thing you expect it to do (AKA debugging)
-2. Checking out the code that makes up other people's web pages to learn how they did a particular thing.
+2. Checking out the code that makes up other people's web pages to learn how they did a levelicular thing.
```

Comment:
I think this is because of search + replace from `part` to `level` so should be `particular`

Reply:
Very much what happened here haha, must've slipped by me.